### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by mhairimca**
Added perpetual contracts

### CODE REVIEW

File: docs/getting-started/welcome.md [hunk 1]
```diff
@@ -1,36 +1,55 @@
...
-
-## For UMA Tokenholders
-
-If you are interested in the UMA project token, please read this [section](uma-tokenholders/uma-holders.md) to learn more about the role of the UMA project token in the DVM and the rights and responsibilities of UMA project tokenholders.
-
-## Contact 
-If you have any questions or want to connect with the UMA community, please check out our community [page](community/community-overview) or join our [Discord](https://discord.umaproject.org/).
+---
+title: Welcome to UMA
+sidebar_label: Welcome to UMA
```
Comment (Tommy1231232) [inline docs/getting-started/welcome.md:3]: Same comment as other PRs

File: docs/synthetic-tokens/perpetuals.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+sidebar_label: Perpetual Contracts
+slug: /More on Priceless Financial Contracts
+---
+
+Perpetual contracts are priceless financial contracts which have no expiry date, but which track tokens through a funding rate swap design.
+
+In contrast to expiring contracts, there is no “expiry” in perpetual contracts. Without the promise of impending expiry, synthetic token holders can no longer assume that their synthetic tokens can be redeemed at some precise time in the future for a predetermined value. Instead a mechanism is introduced to nudge the token to a fair value, through a “funding rate” which forces a payment between token sponsors and holders in order to push the synthetic token’s price in one direction.
+
+When the perpetual contract’s synthetic token is trading “too high” relative to its tracking price, then a payment can be made from holders (who are long the token) to sponsors (who are short the token) to discourage marginal buyers. This “error” between the synthetic tokens’ market value and the current value of the price identifier can be linearly mapped to a “funding rate”, which is a % of each token’s value that is transferred between sponsor and holder.
```
Comment (Tommy1231232) [inline docs/synthetic-tokens/perpetuals.md:10]: We should describe how funding rates are proposed  (who does it, why they do it, what reward they get)

File: docs/getting-started/welcome.md (hunk 1 above)
Comment (mhairimca) [reply docs/getting-started/welcome.md:3]: Welcome file is now overwritten by PR143

File: docs/synthetic-tokens/perpetuals.md (hunk 2 above)
Comment (mhairimca) [reply docs/synthetic-tokens/perpetuals.md:10]: Would that not be covered more in the additional material that is being written on the Perpetual?

I feel that might get a bit technical for this section.

**Comment by smb2796** [comment]
Closing. Will be replaced with updated pr.

**Pull request closed without merging**